content/source/docs/enterprise/admin/upgrades.html.md

```diff
@@ -10,13 +10,38 @@ version. [Learn more about availability during upgrades here](../system-overview
 
 ## Online
 
+### Replicated Administration Console
+
 1. From the installer dashboard (`https://<TFE HOSTNAME>:8800/dashboard`),
     click the "Check Now" button; the new version should be recognized.
 1. Click "View Update".
 1. Review the release notes and then click "Install Update".
 
+### Replicated Command Line Interface
+
+1. Login to a command line terminal of the Terraform Enterprise host machine.
```

Comment:
This is more of a preference thing from me, so feel free to disregard if you disagree.

```suggestion
1. Connect to the Terraform Enterprise host machine using SSH.
```